Loop/Managers/DeviceDataManager.swift

```diff
@@ -115,6 +151,46 @@ final class DeviceDataManager {
         self.pluginManager = pluginManager
         self.alertManager = alertManager
         
+        self.healthStore = HKHealthStore()
```

Comment:
Initializing the stores in DDM so we can do HK authorization with them (and not to have to mock the HKHealth store for testing)